utilities/api/index.spec.ts

```diff
@@ -1,39 +1,39 @@
 import { CONTENTFUL_API_URL } from './constants'
-import { getAllPageContentsWithSlugString, getPageContentBySlugString } from './index'
+import { getAllPathnames, getPageContentByPathname } from './'
 import type { PageContentProps } from '../../components/Main/PageContent'
-import type { PageContentsWithSlugString } from '../../pages/[[...slug]]'
+import type { Pathname } from './'
 import { rest } from 'msw'
 import { setupServer } from 'msw/node'
 import 'whatwg-fetch'
 
-const pageContent: PageContentProps = {
-  content: 'mock content',
-  title: 'mock title',
-}
-const allPageContentsWithSlug: PageContentsWithSlugString[] = [
+const mockAllPathnames: Pathname[] = [
   {
-    slugString: 'demo/js',
+    pathname: 'demo/js',
   },
   {
-    slugString: 'demo/python',
+    pathname: 'demo/python',
   },
 ]
+const mockPageContent: PageContentProps = {
+  content: 'mock content',
+  title: 'mock title',
+}
 
+// todo: use mock graphql
 const server = setupServer(
   rest.post(CONTENTFUL_API_URL, (req, res, ctx) => {
-    const query = (req?.body as Record<string, string>)?.query
-    let items
+    const query = (req.body as Record<string, string>).query
+    let items: PageContentProps[] | Pathname[]
 
-    // todo: use mock graphql
     switch (true) {
-      case /{\s*slugString\s*}/.test(query):
-        items = allPageContentsWithSlug
-        break
       case /{\s*content\s*title\s*}/.test(query):
-        items = [pageContent]
+        items = [mockPageContent]
+        break
```

Comment:
Still have one `let` in the test. This part should be rewritten using mocking GraphQL API from MSW. So this `let` will be removed then. 
But still curious, how to avoid using let in this scenario：
```js
let x
switch (someString) {
  case 'apple':
    x = 1
    break
  case 'banana':
    x = 0
    break
  case 'orange':
    x = 0
    break
}
```